bin/check-consul-servers.rb

```diff
@@ -85,7 +85,7 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
 
   def run
     url = "#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/peers"
-    options = { timeout: config[:timeout],
+    options = { timeout: config[:timeout].to_i,
```

Comment:
This should work but if someone else adds that timeout to something else they need to do it again. I think rather than having to assert it's type where it is used it would be best to assert it as it is being passed in using a `proc` like [this](https://github.com/sensu-plugins/sensu-plugins-redis/blob/2.3.2/bin/check-redis-memory-percentage.rb#L41)